syntax/coq.vim

```diff
@@ -39,6 +39,10 @@ if !exists('b:coqtail_did_highlight') || !b:coqtail_did_highlight
       hi def CoqtailChecked ctermbg=4 guibg=LightGreen
       hi def CoqtailSent ctermbg=7 guibg=LimeGreen
     endif
+    hi def link CoqtailDiffAdded DiffText
+    hi def link CoqtailDiffRemoved DiffDelete
+    hi def link CoqtailDiffAddedBg DiffChange
+    hi def link CoqtailDiffRemovedBg DiffChange
```

Comment:
Should one of these not use `DiffAdd` (`:h hl-DiffAdd` says it's for an added line)? I'm having trouble finding documentation on what each of the Coq diff types means so it's possible these are correct as-is.

Reply:
`DiffAdd` is for highlighting the entirety of a new line, which Coqtop doesn't have a separate kind of highlighting for (it just shows the entire line as `CoqtailDiffAdded`).

I'm using `DiffChange` and `DiffText` because those are respectively meant for the background and content of a changed line, and are thus likely to be defined to work together. I avoided using `DiffChange` and `DiffAdd` because the colors are more likely to clash.

Reply:
Actually, `CoqtailDiffRemovedBg` should also be `DiffDelete`. Otherwise it's impossible to tell what's going on..

Reply:
See [comment above](https://github.com/whonore/Coqtail/pull/169#discussion_r550818445) for what that looks like now.